player/src/App.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Typography, Container, makeStyles, Paper, Grid, Button } from '@material-ui/core';
+import { Synth } from "tone";
+
+const useStyles = makeStyles((theme)=>({
+  root: {
+    width: '100%',
+    maxWidth: 500,
+    flexGrow: 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  },
+}));
+
+var synth = new Synth().toMaster();
+
+function handleClickQ(e) {
+  e.preventDefault();
+  console.log('The link was clicked.');
```

Comment:
Try to remember to remove console.log()'s unless they are required.  It will just clean up the code and make it more manageable.